### PR TITLE
docs: trans of getting start

### PR DESCRIPTION
**Pull request by Qianmoxsn**
- translate `draw_cube.md`
- translate `initialization.md`
- translate `load_model.md`
- translate `script.md`
`script.md` contains link to [Life Cycle](/guide/core/component#life-cycle)
- so translate #Life Cycle of `component.md`

@lslzl3000 @Noah133552

### CODE REVIEW

File: docs/guide/getting_start/draw_cube.md [hunk 1]
```diff
@@ -1,100 +1,111 @@
...
 
 <Demo src="/demos/getting_start/cube.ts"></Demo>
 
 <<< @/public/demos/getting_start/cube.ts
 
-## 引入模块
-首先，我们需要引入相应的模块：
+## import modules
+
+First, we need to import the corresponding modules we need:
```
Comment (lslzl3000) [inline docs/guide/getting_start/draw_cube.md:11]: we need ..... we need ?

File: docs/guide/getting_start/script.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
-    // update 生命周期，主循环每一帧执行
-  }
+    public update() {
+        //update lifecycle, executed every frame in the main loop
+    }
 }
 ```
 
-我们可以通过 `this.object3D` 获取当前组件挂载到的 [object3D](/guide/core/object) 对象，从而更改节点的状态。例如，在 `update` 中更新 `object3D.rotationY`, 即每一帧围绕 `Y` 轴旋转加 `1` 度。
+We can get the current [object3D](/guide/core/object) object that the component is mounted on through `this.object3D`, and then change the state of the node. For example, in `update`, update `object3D.rotationY`, that is, rotate around the `Y` axis by `1` degree every frame.
```
Comment (lslzl3000) [inline docs/guide/getting_start/script.md:22]: It seems too complex,  not translate word by word, try to improve it?

**Review by lslzl3000 — APPROVED** [review]

File: docs/guide/getting_start/draw_cube.md [hunk 3]
```diff
@@ -1,100 +1,111 @@
...
+
+In this section, we will take a quick look at the flow of the engine through an example of drawing a cube:
 
 <Demo src="/demos/getting_start/cube.ts"></Demo>
 
 <<< @/public/demos/getting_start/cube.ts
 
-## 引入模块
-首先，我们需要引入相应的模块：
+## import modules
```
Comment (Noah133552) [inline docs/guide/getting_start/draw_cube.md:9]: import-->Import

File: docs/guide/core/component.md [hunk 4]
```diff
@@ -69,18 +69,18 @@ let light = obj.getComponent(DirectLight)
...
+|    Cycle     | Description                                                                   |
+|:------------:|-------------------------------------------------------------------------------|
+|     init     | Component initialization, only run once when the component is created         |
+|    start     | Component start running, only called before the first `onUpdate`              |
+|    update    | Component update every frame                                                  |
+| beforeUpdate | Component update before rendering                                             |
+|  lateUpdate  | Component update after rendering                                              |
+|   onEnable   | Component `enable = true` callback, component join the main loop update       |
+|  onDisable   | Component `enable = false` trigger, component will no longer trigger the loop |
+|     stop     | Component is destroyed before being called                                    |
```
[the remark this answers is not in the document]
Comment (Qianmoxsn) [reply docs/guide/core/component.md:84]: this part will fix by PR:#128

File: docs/guide/getting_start/draw_cube.md [hunk 5]
```diff
@@ -1,100 +1,112 @@
...
 let scene3D = new Scene3D();
 ```
 
-## 添加摄像机控件
+## Add camera controller component
+
 ```ts
-// 新建摄像机实例
-let cameraObj:Object3D = new Object3D();
+// Creat a camera object
```
Comment (Noah133552) [inline docs/guide/getting_start/draw_cube.md:56]: Create， spelling mistake

File: docs/guide/getting_start/initialization.md [hunk 6]
```diff
@@ -1,38 +1,38 @@
-# 初始化引擎
-## 创建 Engine3D 实例
-在使用引擎前，必须通过 `Engine3D.init()` 方法进行初始化，引擎会自动创建一个 `Engine3D` 实例用于后续的操作
+# Initialization
+## Creat instances of Engine3D
```
Comment (Noah133552) [inline docs/guide/getting_start/initialization.md:2]: Create， spelling mistake

**Review by Noah133552 — APPROVED** [review]

**Pull request merged**